src/lib/viewers/media/DashViewer.js

```diff
@@ -488,17 +486,68 @@ class DashViewer extends VideoBaseViewer {
             textCues.push(new shaka.text.Cue(start, end, text));
         });
 
+        return textCues;
+    }
+
+    /**
+     * Loads auto-generated captions
+     *
+     * @public
+     * @param {Object} [skillsData] - optional skills data used to generate captions
+     * @return {void}
+     */
+    loadAutoGeneratedCaptions(skillsData) {
+        const skillsCards = skillsData || getProp(this.options, 'file.metadata.global.boxSkillsCards.cards');
+        // Do nothing if we don't have skills data or already have captions
+        if (!skillsCards || this.autoCaptionDisplayer) {
+            return;
+        }
+
+        // Save passed in skills data to avoid regenerating captions unnecessarily
+        this.skillsData = skillsData;
+
+        const textCues = this.parseTextCues(skillsCards);
+
         if (textCues.length > 0) {
             this.autoCaptionDisplayer = new shaka.text.SimpleTextDisplayer(this.mediaEl);
```

Comment:
Could you reuse this, if it's already been created beforehand? 
IE) `this.autoCaptionDisplayer = this.autoCaptionDisplayer || new SimpleTextDisplayer(...)`